### ISSUE TITLE
Add support for macOS Monterey

### ISSUE THREAD

**Issue by AmpCat** [sudowork/fix_m1_rgb#4]
Yes, I know it's only just been made available to developers, but at this point the script just says 'OS not supported' so I can't even try to see if it works. I actually upgraded to the beta to see if it might fix this issue once and for all. Not yet.

**Comment by sudowork** [comment]
Thanks @AmpCat. Just added support for this. When running the script without the `--dry-run` flag, it will warn and prompt you if you want to continue. Just type a `y` and hit return to continue anyways. In `--dry-run` mode, it will just proceed.

![image](https://user-images.githubusercontent.com/1113327/121941005-98f94100-cd03-11eb-9f9c-0fdbcb77d3fe.png)

Let me know how it goes please - would love to hear back and see if it worked for you!

**Comment by AmpCat** [comment]
Awesome! Won't be able to try it until tomorrow. I appreciate it. Hopefully this will finally fix the issue for me.

**Comment by sudowork** [comment]
Any luck @AmpCat ?

**Comment by AmpCat** [comment]
Sorry for the delay there, work has kept me real busy. Tried it a few times, and still not getting any results. I tried it before moving to macOS 12 as well, and didn't have any luck there.

However, I'm seeing a few errors:
Did not fix Display 1D127834-08B9-C782-0857-D6964E3302DB. Values for PixelEncoding and Range not as expected.
Did not fix Display 19440EC3-57A8-E480-3D19-FC1626C9B248. Values for PixelEncoding and Range not as expected.
Skipping `/Users/robert/Library/Preferences/ByHost/com.apple.windowserver.displays.E5CAC908-AABD-5231-90D4-85A96B3DD38C.plist`. No `LinkDescription` found in display config. Try rotating your display from Display settings to generate the field in the plist.

I rotated the display as instructed, but it found a lot of displays, so hard for me to know which it was complaining about.

Let me know if you want the full log
